stubs/networkx/METADATA.toml

```
@@ -0,0 +1,4 @@
+version = "3.1"
+upstream_repository = "https://github.com/networkx/networkx"
+requires = ["pandas", "numpy", "scipy", "pygraphviz"]
```

It seems that `pygraphviz` is having issues being installed and compiled during testing.
You can probably remove it from here for now, and remove the import from `stubs/networkx/networkx/drawing/nx_agraph.pyi` for now.

It also seems intended to be optional, so probably a good to not require it https://github.com/networkx/networkx/blob/main/pyproject.toml#L88
```suggestion
requires = ["pandas", "numpy", "scipy"]
```